Formula/mabel.rb

```diff
@@ -0,0 +1,28 @@
+class Mabel < Formula
+  desc "Fancy BitTorrent client for the terminal"
+  homepage "https://github.com/smmr-software/mabel"
+  url "https://github.com/smmr-software/mabel.git",
+    tag:      "v0.1.1",
+    revision: "23490e76aea68357b224dd4d7f1aed822ce2cd46"
+  license "GPL-3.0-or-later"
+  head "https://github.com/smmr-software/mabel.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.commit=#{Utils.git_head}
+      -X main.builtBy=homebrew
+      -X main.date=#{time.iso8601}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+
+  test do
+    vrsn_out = shell_output("#{bin}/mabel --version")
+    assert_match "Mabel #{version}", vrsn_out
+    assert_match "Built by: homebrew", vrsn_out
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

We can keep this test to make sure that `main.version` works, but we need an additional test to make sure what we installed is working properly.